src/RoslynPad.Hosting/ExecutionHost.cs

```diff
@@ -156,14 +159,18 @@ private static DelegatingTextWriter CreateConsoleWriter()
 
         public ExecutionHost(string hostPath, string initialWorkingDirectory,
```

Comment:
They are just three primitives with default values. Maybe if we wanted to change all other settings a proxy to `CSharpCompilationOptions` would be needed, but in the context only the three of the all options are practically relevant.